packages/cli/src/packages-to-be/address-resolver.ts

```diff
@@ -0,0 +1,30 @@
+import { registryABI } from '@celo/abis'
+import { NULL_ADDRESS, StrongAddress } from '@celo/base'
+import { getClient } from './client'
+
+export const REGISTRY_CONTRACT_ADDRESS = '0x000000000000000000000000000000000000ce10'
+
+const cache: Record<string, StrongAddress> = {}
+
+// TODO should use the enum for contract names?
```

Comment:
there is something about using `type "Account" | "Validator" | "LockedCelo"` that appeals to me 

no import of an enum needed, vs code autocompletes anyway. and its closer to what is sent to contract. 

what do you think?